test/test_transforms.py

```diff
@@ -1482,9 +1482,13 @@ def test_five_crop(single_dim):
 
 @pytest.mark.parametrize('policy', transforms.AutoAugmentPolicy)
 @pytest.mark.parametrize('fill', [None, 85, (128, 128, 128)])
-def test_autoaugment(policy, fill):
+@pytest.mark.parametrize('grayscale', [True, False])
+def test_autoaugment(policy, fill, grayscale):
     random.seed(42)
     img = Image.open(GRACE_HOPPER)
+    if grayscale:
+        img = img.convert('L')
+        fill = (fill[0], ) if isinstance(fill, tuple) else fill
```

Comment:
Should we move this to a private util method to avoid code duplication?

Reply:
I suggest defining an util function at the top of the file as follow:

```python
def get_grayscale_test_image(img, fill=None):
    img = img.convert('L')
    fill = (fill[0], ) if isinstance(fill, tuple) else fill
    return img, fill
```
And, for example, update those test cases as below:
```python
@pytest.mark.parametrize('num_ops', [1, 2, 3])
@pytest.mark.parametrize('magnitude', [7, 9, 11])
@pytest.mark.parametrize('fill', [None, 85, (128, 128, 128)])
@pytest.mark.parametrize('grayscale', [True, False])
def test_randaugment(num_ops, magnitude, fill, grayscale):
    random.seed(42)
    img = Image.open(GRACE_HOPPER)
    img, fill = get_grayscale_test_image(img, fill)  # change here
    transform = transforms.RandAugment(num_ops=num_ops, magnitude=magnitude, fill=fill)
    for _ in range(100):
        img = transform(img)
    transform.__repr__()
```

What do you think?

Reply:
Sounds good to me. Just make it private by prepending `_` to it. :)